### DISCUSSION TITLE
How do I turn a JSON object into a MikroORM entity object and give it to UnitOfWork to manage.

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2383] [Q&A] [answered]
How do I turn a JSON object into a MikroORM entity object and give it to UnitOfWork to manage.

I'm working on a database object cache, but after reading from the Redis cache, I can't serialize it to the stored value.

**Comment by B4nan** [comment, marked as answer]
`em.merge()` or `em.map()` (map handles also conversion of database column names to property names, you might not need that here)

**Reply by Lydanne** [reply]
Let me see

**Reply by Lydanne** [reply]
I have achieved it
